recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -202,6 +209,50 @@ def __del__(self):
             # created in "/tmp".
             self.driver.quit()
 
+    def startChrome(self, width, height, env):
+        """
+        Starts a Chrome instance. Will use Chromium is Google Chrome is not installed.
+        :param width: the width of the browser window.
+        :param height: the height of the browser window.
+        :param env: the environment variables, including the display to start
+                    the browser in.
+        """
+
+        options = ChromeOptions()
+
+        # "webSocketUrl" is needed for BiDi; this should be set already by
+        # default, but just in case.
```

Comment:
It seems that `webSocketUrl` is set by default in Firefox, but not in Chromium, so the comment should be only `"webSocketUrl" is needed for BiDi`.